api/Document.json

```diff
@@ -8901,6 +8901,53 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "accepts_TrustedHTML": {
+          "__compat": {
+            "description": "Accepts `TrustedHTML` instances",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/write",
```

Comment:
Let's only add `mdn_url` if it describes this feature (here: accepting TrustedHTML).

```suggestion
```